modules/upgrade/pages/db-migration-12.adoc

```diff
@@ -0,0 +1,123 @@
+[[db-migration-12]]
+= Database Migration from Version 10 to 12
+
+This section covers upgrading the PostgreSQL database from version{nbsp}10 to version{nbsp}12.
+If you are already using PostgreSQL 12, you do not need to perform this migration.
+If you are using an older version, such as version 9.6, see xref:upgrade:db-migration.adoc[].
+
+If you want to upgrade to the latest {productname} version, you must be using PostgreSQL version 10 or 12.
```

Comment:
Not sure about this sentence. Was it a copy from the last migration? 

As with 4.0 PG 10 is the only option, You can only migrate after you switched to SUMA4.1 .

Reply:
I remove the confusing sentence.